rubygem-sprockets/rubygem-sprockets.spec

```diff
@@ -0,0 +1,81 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name sprockets
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 3.5.2
+Release: 1%{?dist}
+Summary: Rack-based asset packaging system
+Group: Development/Languages
+License: MIT
+URL: https://github.com/rails/sprockets
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ror}rubygem(rack) > 1
+Requires: %{?scl_prefix_ror}rubygem(rack) < 3
+Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) >= 1
```

Comment:
This isn't in the ror collection, but we do happen to have it in the plugins repo.  It'll need moving (update tito props and comps/) and to use scl_prefix here.